### ISSUE TITLE
[NS] CSS customisable

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/GeoNature-citizen#36]
Il faudrait avoir la possibilité de customiser l'interface de GeoNature-citizen, a minima les couleurs. par exemple par l'utilisation de variables comme dans cet exemple: https://coryrylan.com/blog/theming-angular-apps-with-css-custom-properties

**Comment by camillemonchicourt** [comment]
Oui essentiel.
Sinon comme on fait dans les autres projets : un CSS custom vide mais appelé après les autres permettant de surcoucher les classes par défaut.

Voir GN-atlas

**Comment by samuelpriou** [comment]
Oui, il faut effectivement que les structures utilisatrices de GeoNature-citizen puissent customiser l'interface avec leurs couleurs et autres logos et images.
Dans GN-atlas ces fichiers sont stockés dans static/custom

**Comment by camillemonchicourt** [comment]
Juste charger un CSS custom vide par défaut, que chacun peut utiliser pour surcoucher des classes suffit.

**Comment by patkap** [comment]
@lpofredc , la pr #84 couvre-t'elle tous les besoins évoqués ?

**Comment by patkap** [comment]
> @lpofredc , la pr #84 couvre-t'elle tous les besoins évoqués ?

La fonctionnalité a été réintégrée (#oops), c'est génial ... il nous faut un stealth mode !

**Comment by lpofredc** [comment]
Surcouche custom.scss

**Comment by jbdesbas** [comment]
Salut,
Petit souci il me semble, les css des composants sont chargés tout à la fin, et surcouchent donc le _custom.css_ . Je ne sais pas comment on peut solutionner ça ?

**Comment by lpofredc** [comment]
Etrange, en principe, il est chargé à la fin. cf. `angular.json`
https://github.com/PnX-SI/GeoNature-citizen/blob/14301e14db14cd3246cb8944ca288435cd710cd3/frontend/angular.json#L31-L33

**Comment by jbdesbas** [comment]
Là il me semble que ce n'est ce qui est dans _style.css_ qui pose problème, mais ce qui est défini au niveau des components.
![image](https://user-images.githubusercontent.com/6163107/67871779-2f79ee00-fb31-11e9-8b71-17759cc29fea.png)

Ou alors il faut faire 2 css pour chaque component (1 base + 1 custom) ?
